mturk/mturk.ipynb

```diff
@@ -63,44 +76,61 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import boto3\n",
-    "import json\n",
-    "import copy\n",
-    "\n",
-    "USING_PROD = None\n",
-    "EXTERNAL_SUBMIT = None\n",
-    "\n",
-    "# Read config and extract relevant settings\n",
-    "with open(CONFIG_PATH, 'r') as f:\n",
-    "    config = json.loads(f.read())\n",
-    "    \n",
-    "hit_config = config['hitCreation']\n",
-    "\n",
-    "EXTERNAL_SUBMIT = config['advanced']['externalSubmit']\n",
+    "# Safety flags that prevent you from accidentally messing up your HITs. \n",
+    "# Set to False except when you are performing these specific tasks.\n",
+    "ALLOW_HIT_CREATION = False\n",
+    "ALLOW_ASSIGNMENT_ADDITION = False\n",
+    "ALLOW_CREATE_QUAL = False\n",
+    "ALLOW_UPDATE_EXPIRATION = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Sandbox or Production? You only spend money in Production.\n",
+    "USING_PROD = False\n",
```

Comment:
Could you please change this back so that this pulls from the config instead of explicitly setting it here? If you configure your task as an iFrame with MTurk as the back-end, it uses the config var to figure out what url to post data to in order to get it back to MTurk. I like to keep this variable in just one place (the config) because otherwise you can get into a situation where workers can't submit their HIT and they get mad. 

There is a way to avoid this, but it's outside the scope of this PR--I will add it in after this is submitted.

Reply:
Ok now I understand why the config was in the root. I did a search for ['production'] but didn't think a JS file would be using it. My reasoning was that whether the HIT is in sandbox or production is outside the properties of the HIT itself but I will change it back for now.